admin/lib/validator/new-check-window/new-check-window-familiarisation-check-end-date-validator.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+const moment = require('moment')
+
+const checkWindowErrorMessages = require('../../errors/new-check-window')
+const dateService = require('../../../services/date.service')
+const XRegExp = require('xregexp')
+const { isEmpty, isInt } = require('validator')
+
+/**
+ * Validates check window familiarisation check end date data
+ * @param {Object} validationError
+ * @param {Object} familiarisationCheckEndDateData
+ */
+module.exports.validate = (validationError, familiarisationCheckEndDateData) => {
```

Comment:
could you utilise moment here for checking valid dates?  not every month has 31 days....

Reply:
@GuyHarwood that's an excellent point. I have updated the date validators to include this.